pkg/binding/definition.go

```diff
@@ -65,6 +67,13 @@ func (d *stringDefinition) Apply(u *unstructured.Unstructured) (Value, error) {
 	if d.outputName == "" {
 		return nil, fmt.Errorf("cannot use generic service.binding annotation for string elements, need to specify binding key like service.binding/foo")
 	}
+	if d.value != "" {
+		return &value{
+			v: map[string]interface{}{
+				d.outputName: reflect.ValueOf(d.value),
+			},
+		}, nil
+	}
```

Comment:
Can we get a test for this?  `pkg/binding/definition_test.go` doesn't cover this, since it doesn't have a test for a set `value` getting set in a `stringDefinition`.